src/utils/config.ts

```diff
@@ -26,7 +27,7 @@ export type RedisParams = t.TypeOf<typeof RedisParams>;
 export const SpidParams = t.intersection([
   t.interface({
     AUTH_N_CONTEXT: NonEmptyString,
-
+    ENABLE_JWT: t.boolean,
```

Comment:
This prop can be remove because is defined in `JWTParams `
```suggestion
```

Reply:
You're right :D